tests/unit/iam/test_policy.py

```diff
@@ -0,0 +1,680 @@
+                         'true')
+
+
+class TestDeletePolicy(AWSMockServiceTestCase):
+    connection_class = IAMConnection
+
+    def default_body(self):
+        return b"""
+<DeletePolicyResponse xmlns="https://iam.amazonaws.com/doc/2010-05-08/">
+  <ResponseMetadata>
+    <RequestId>4706281b-3d19-11e4-a4a0-cffb9EXAMPLE</RequestId>
+  </ResponseMetadata>
+</DeletePolicyResponse>
+        """
+
+    def test_delete_policy(self):
+        self.set_http_response(status_code=200)
+        response = self.service_connection.delete_policy(
+                'arn:aws:iam::123456789012:policy/S3-read-only-example-bucket')
+
+        self.assert_request_parameters(
+            {'Action': 'DeletePolicy',
+             'PolicyArn': 'arn:aws:iam::123456789012:policy/S3-read-only-example-bucket'},
+            ignore_params_values=['Version'])
+
+        self.assertEqual('request_id' in response['delete_policy_response']
+                                                 ['response_metadata'],
+                         True)
+
+
+class TestDeletePolicyVersion(AWSMockServiceTestCase):
+    connection_class = IAMConnection
+
+    def default_body(self):
+        return b"""
+<DeletePolicyVersionResponse xmlns="https://iam.amazonaws.com/doc/2010-05-08/">
+  <ResponseMetadata>
+    <RequestId>268e1556-3d19-11e4-a4a0-cffb9EXAMPLE</RequestId>
+  </ResponseMetadata>
+</DeletePolicyVersionResponse>
+        """
+
+    def test_delete_policy_version(self):
+        self.set_http_response(status_code=200)
+        response = self.service_connection.delete_policy_version(
+                'arn:aws:iam::123456789012:policy/S3-read-only-example-bucket',
+                'v1')
+
+        self.assert_request_parameters(
+            {'Action': 'DeletePolicyVersion',
+             'PolicyArn': 'arn:aws:iam::123456789012:policy/S3-read-only-example-bucket',
+             'VersionId': 'v1'},
+            ignore_params_values=['Version'])
+
+        self.assertEqual('request_id' in response['delete_policy_version_response']
+                                                 ['response_metadata'],
+                         True)
+
+
+class TestGetPolicy(AWSMockServiceTestCase):
+    connection_class = IAMConnection
+
+    def default_body(self):
+        return b"""
+<GetPolicyResponse xmlns="https://iam.amazonaws.com/doc/2010-05-08/">
+  <GetPolicyResult>
+    <Policy>
+      <PolicyName>S3-read-only-example-bucket</PolicyName>
+      <DefaultVersionId>v1</DefaultVersionId>
+      <PolicyId>AGPACKCEVSQ6C2EXAMPLE</PolicyId>
+      <Path>/</Path>
+      <Arn>arn:aws:iam::123456789012:policy/S3-read-only-example-bucket</Arn>
+      <AttachmentCount>9</AttachmentCount>
+      <CreateDate>2014-09-15T17:36:14Z</CreateDate>
+      <UpdateDate>2014-09-15T20:31:47Z</UpdateDate>
+    </Policy>
+  </GetPolicyResult>
+  <ResponseMetadata>
+    <RequestId>684f0917-3d22-11e4-a4a0-cffb9EXAMPLE</RequestId>
+  </ResponseMetadata>
+</GetPolicyResponse>
```

Comment:
This response could contain a ``<Description>`` element.  Might want to include it here or test both cases.